diff --git a/node_modules/next/dist/build/webpack/plugins/next-font-manifest-plugin.js b/node_modules/next/dist/build/webpack/plugins/next-font-manifest-plugin.js
index 1752d5b..7c46a6f 100644
--- a/node_modules/next/dist/build/webpack/plugins/next-font-manifest-plugin.js
+++ b/node_modules/next/dist/build/webpack/plugins/next-font-manifest-plugin.js
@@ -57,7 +57,7 @@ class NextFontManifestPlugin {
                     // After all modules are created, we collect the modules that was created by next-font-loader.
                     (0, _utils.traverseModules)(compilation, (mod, _chunk, chunkGroup)=>{
                         var _mod_request;
-                        if (mod == null ? void 0 : (_mod_request = mod.request) == null ? void 0 : _mod_request.includes('/next-font-loader/index.js?')) {
+                        if (mod == null ? void 0 : (_mod_request = mod.request) == null ? void 0 : _mod_request.replace(/\\/g, '/').includes('/next-font-loader/index.js?')) {
                             var _mod_buildInfo;
                             if (!((_mod_buildInfo = mod.buildInfo) == null ? void 0 : _mod_buildInfo.assets)) return;
                             const chunkEntryName = (appDirBase + chunkGroup.name).replace(/[\\/]/g, _path.default.sep);
